File: Data Structures & Algorithms/two-integer-sum/submission-5.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        num_set = {}
        for i, n in enumerate(nums):
            diff = target - n
            if diff in num_set:
                return [num_set[diff], i]
            num_set[n] = i

File: Data Structures & Algorithms/remove-element/submission-5.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int l = 0;
        int r = nums.size() - 1;
        while (l <= r) {
            if (nums[l] == val) {
                int temp = nums[l];
                nums[l] = nums[r];
                nums[r] = temp;
                r -= 1;
            }
            else {
                l += 1;
            }
        }   
        return l;
    }
};